src/logic/currencyValues/store/actions/setCurrencySelected.js

```diff
@@ -9,5 +9,8 @@ export const SET_CURRENT_CURRENCY = 'SET_CURRENT_CURRENCY'
 // eslint-disable-next-line max-len
 export const setCurrencySelected = createAction<string, *>(
   SET_CURRENT_CURRENCY,
-  (currencyValueSelected: $Keys<typeof AVAILABLE_CURRENCIES>): CurrencyValuesProps => ({ currencyValueSelected }),
+  (safeAddress: string, currencyValueSelected: $Keys<typeof AVAILABLE_CURRENCIES>): CurrencyValuesProps => ({
+    safeAddress,
+    currencyValueSelected,
```

Comment:
same rule applies to this one `currencyValueSelected` -> `selectedCurrencyValue` or `selectedCurrency`